shared/teams/team/container.js

```diff
@@ -197,9 +197,13 @@ const mergeProps = (stateProps, dispatchProps, ownProps) => {
     customComponent,
     headerStyle: {borderBottomWidth: 0},
     invites: stateProps._invites.toJS(),
-    members: stateProps._memberInfo
-      .toArray()
-      .sort((a: Constants.MemberInfo, b: Constants.MemberInfo) => a.username.localeCompare(b.username)),
+    members: stateProps._memberInfo.toArray().sort((a: Constants.MemberInfo, b: Constants.MemberInfo) => {
+      if (you && !youImplicitAdmin) {
+        if (a.username === you) return -1
```

Comment:
subtle flaw here...if `a.username === you` and `b.username === you`, it should return 0.

Since `_memberInfo` is a set, probably clearer to remove `you` if necessary, convert to an array and sort normally, then add `you` back in the right position. What do you think?